### PR TITLE
Save and read pictures from db

### PR DESCRIPTION
**Pull request by Bradford-Wong**
I branched off of Brandon's work, so a lot of the "new code" is actually just from his branch. Anyways, I made it so that users can save pictures to the database and then see those pictures on the detailed submission screen. There is work that we can do to clean up the screens and make things more efficient, but this should work for now. We can work on optimizations later.

### CODE REVIEW

File: app/src/main/java/com/myvetpath/myvetpath/AddPicturesActivity.java [hunk 1]
```diff
@@ -215,31 +316,80 @@ public void onClick(DialogInterface arg0, int arg1) { //open up the gallery
                                 pictureActionIntent,
                                 GALLERY_PICTURE);
 
-
                     }
                 });
 
         myAlertDialog.setNegativeButton("Camera",
```
Comment (jeffreykat) [inline app/src/main/java/com/myvetpath/myvetpath/AddPicturesActivity.java]: I'm not sure if it's the emulator or something else but when I select the camera option and take the picture it does not go back to the app. I have to close the camera, retry adding a picture, and get it from the gallery.

File: app/src/main/java/com/myvetpath/myvetpath/AddPicturesActivity.java [hunk 2]
```diff
@@ -215,31 +316,80 @@ public void onClick(DialogInterface arg0, int arg1) { //open up the gallery
...
 
+
+    private File createImageFile() throws IOException {
+        // Create an image file name
+        String timeStamp = new SimpleDateFormat("yyyyMMdd_HHmmss").format(new Date());
+        String imageFileName = "JPEG_" + timeStamp + "_";
+        imageNames[selectedImageView] = imageFileName + ".jpg";
+
+
+        File storageDir = getExternalFilesDir(Environment.DIRECTORY_PICTURES);
```
Comment (jeffreykat) [inline app/src/main/java/com/myvetpath/myvetpath/AddPicturesActivity.java]: It seems like this is storing the image in Pictures, not the Gallery. This isn't necessarily a problem but we should make a note of it in the info activity so users don't get lost.

File: app/src/main/java/com/myvetpath/myvetpath/Client.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    public static final String COLUMN_ID = "ClientID";
+    public static final String COLUMN_FIRSTNAME = "FirstName";
+    public static final String COLUMN_LASTNAME = "LastName";
+    public static final String COLUMN_PHONENUMBER = "PhoneNumber";
+    public static final String Column_EMAIL = "Email";
+
+    //Create Table String
+    public static final String CREATE_TABLE =
+            "CREATE TABLE " + TABLE_NAME + "(" + COLUMN_ID +
+                    "INTEGER PRIMARY KEY, "
```
Comment (jeffreykat) [inline app/src/main/java/com/myvetpath/myvetpath/Client.java]: I think this should be fixed in Brandon's pull but there has to be a space before INTEGER PRIMARY KEY.

File: app/src/main/java/com/myvetpath/myvetpath/SubDetailsActivity.java [hunk 4]
```diff
@@ -53,6 +60,34 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         TextView internalIDText = findViewById(R.id.subInternalID);
         internalIDText.setText(Integer.toString(internalId));
+
+        //set images
```
Comment (jeffreykat) [inline app/src/main/java/com/myvetpath/myvetpath/SubDetailsActivity.java]: Great job here! Images look good on the screen! Later we might need to make sure images are oriented correctly or allow the user to open them in the gallery to zoom and save them. But that can be done later.

File: app/src/main/java/com/myvetpath/myvetpath/AddPicturesActivity.java (hunk 1 above)
Comment (Bradford-Wong) [reply app/src/main/java/com/myvetpath/myvetpath/AddPicturesActivity.java]: That is strange... it wasn't like this before, but I started trying this again. Sometimes it happens and sometimes it doesn't. We may want to revisit this again later to be safe. Good catch!

File: app/src/main/java/com/myvetpath/myvetpath/SubDetailsActivity.java (hunk 4 above)
Comment (Bradford-Wong) [reply app/src/main/java/com/myvetpath/myvetpath/SubDetailsActivity.java]: Yup I agree. There are definitely some ways to enhance the user experience regarding pictures.

**Pull request merged**